src/reporting/parameter.ts

```diff
@@ -69,10 +69,18 @@ export function createStackParametersQuickSight(scope: Construct, paramGroups?:
     default: 'Comma Delimited Subnet Ids',
   };
 
-  const quickSightPrincipalParam = new CfnParameter(scope, 'QuickSightPrincipalParam', {
+  const quickSightOwnerPrincipalParam = new CfnParameter(scope, 'QuickSightOwnerPrincipalParam', {
     description: 'Arn of the QuickSight principal, QuickSight resource will be owned by this principal.',
     type: 'String',
   });
+  labels[quickSightOwnerPrincipalParam.logicalId] = {
+    default: 'QuickSight Owner Principal Arn',
+  };
+
+  const quickSightPrincipalParam = new CfnParameter(scope, 'QuickSightPrincipalParam', {
```

Comment:
Let's add the pattern to require a valid QuickSight ARN.

Reply:
fixed